egasub/submission/submit.py

```diff
@@ -172,7 +172,8 @@ def submit_dataset(ctx, dry_run=True):
                             Attribute('_submitted_using', 'egasub %s' % ver),
                             Attribute('_icgc_project_code', ctx.obj['SETTINGS']['icgc_project_code']),
                             Attribute('_icgc_project_url', 'https://dcc.icgc.org/projects/%s' % ctx.obj['SETTINGS']['icgc_project_code'])
-                        ]
+                        ],
+                        prompt('Enter a dataset description')
```

Comment:
Can you please change the prompt to:

`Enter a dataset description (3-4 sentences to describe the dataset, may include number of samples, file type and technology/experimentation used)`